evaluate/report/collection.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // AssessmentPerLanguagePerModel holds a collection of assessments per language and model.
-type AssessmentPerLanguagePerModel map[language.Language]map[model.Model]metrics.Assessments
+type AssessmentPerLanguagePerModel map[language.Language]AssessmentPerModel
```

Comment:
This commit is only about language (since model worked before). Plz adapt the commit msg.